src/index.html

```diff
@@ -84,6 +86,8 @@ <h1>Settings</h1>
           <div class="form-group">
             <label for="dialectSelection">Default Gherkin Dialect:</label>
             <select id="dialectSelection" class="form-control" onchange="changeGherkinDialect()">
+              <br/>
+            <input type="checkbox" id="darkMode"><label for="darkMode"> Dark Mode </label>
```

Comment:
this needs to get set to still be set with the value on startup, as if you start up with dark mode on the OS, it doesn't show it as on, so checking dark mode makes it white (inverted), which i'm not a huge fan of

Reply:
also, this is inside the select, it should be outside of the select